### ISSUE TITLE
Suggestion: add uBOPa to uBlock Origin default filterlists

### ISSUE THREAD

**Issue by MasterKia** [nimasaj/uBOPa#53]
TL;DR : Is there a possibility that you could ask the uBlock Origin dev to add uBOPa as one of the persian lists in uBO regional lists besides the IRN-Adblock-Iran?
I found two Poland and Spanish lists there, so I think he wouldn't mind adding a second persian list.
Related issue: `https://github.com/gorhill/uBlock/issues/1112`

Currently the [`IRN: Adblock-Iran`](https://github.com/farrokhi/adblock-iran) is the only default Persian filterlist in uBO.

1) It's a pretty solid list but it doesn't cover some ads that uBOPa block.

2) The maintainer doesn't update the list frequently (as of writing it's 3 months old) whereas the uBOPa gets updated more often.

3) There are open [issues](https://github.com/farrokhi/adblock-iran/issues) that are 3 months old.

**Comment by nimasaj** [comment]
Thanks for the suggestion. 
I leave it to the users to ask uBO to add this list to its regional section if they find it useful.

**Comment by MasterKia** [comment]
> users ask uBO to add this list

Alright, I'd see if I could do that.

**Comment by MasterKia** [comment]
Let's see how this goes: https://www.reddit.com/r/uBlockOrigin/comments/ump0kk/current_persian_filterlist_is_unmaintained/

**Comment by MasterKia** [comment]
Please check: `https://github.com/uBlockOrigin/uAssets/issues/13253#issuecomment-1127958780`

**Comment by MasterKia** [comment]
What's your opinion on this?

`https://github.com/uBlockOrigin/uAssets/discussions/13259#discussioncomment-2768336`
`https://github.com/uBlockOrigin/uAssets/discussions/13259#discussioncomment-2768743`
